### DISCUSSION TITLE
Make players visible online for friends

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17985] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/40531871/165291032-97a4dc05-1425-4deb-b5b3-4318a1e97e82.png) 😞

[This](https://github.com/ppy/osu/issues/12635) is probably related?

**Comment by frenzibyte** [comment, marked as answer]
See https://github.com/ppy/osu/discussions/16395#discussioncomment-1944539
